pkgs/development/arduino/arduino-core/default.nix

```diff
@@ -11,7 +11,8 @@
 , ncurses
 , readline
 , withGui ? false
-, gtk2 ? null
+, gtk3 ? null
+, wrapGAppsHook ? null
```

Comment:
Why is this null by default? It should be in the standard nixpkgs set.

Reply:
I followed the approach taken for the gtk3 dependency, but I'm OK with removing the default value as well